recipes/easysnowdata/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "easysnowdata" %}
+{% set version = "0.0.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/easysnowdata-{{ version }}.tar.gz
+  sha256: 41de137a200a0732370772f66c12ca19f2dac97097d3e4748c23497680cf964b
+
+build:
+  noarch: python
```

Comment:
Please look at the docs for `noarch: python` recipes:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

specifically the use of `{{ python_min }}`. This will fix up your linting errors.